client/asset/eth/nodeclient.go

```diff
@@ -513,7 +518,7 @@ func (n *nodeClient) transactionConfirmations(ctx context.Context, txHash common
 	// tx pool, and when our peers are ready to supply the info. I saw a
 	// CoinNotFoundError in TestAccount/testSendTransaction, but haven't
 	// reproduced.
-	return 0, asset.CoinNotFoundError
+	return 0, fmt.Errorf("cannot find %v: %w", txHash, asset.CoinNotFoundError)
```

Comment:
I see this randomly hit sometimes, but other nodes do see the tx on the other nodes. I don't know why it's being missed.